tests/typescript/classes/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -87,5 +106,36 @@ export class VisTimelineComponent2
     OnDestroy,
     AndSomethingReallyReallyLong {}
 
+class loooooooooooooooooooong
+  extends looooooooooooooooooong
+  implements loooooooooooooooooooong
+{
+  // leading comment
+  property: string;
+}
+
+class loooooooooooooooooooong
+  extends looooooooooooooooooong
+  implements loooooooooooooooooooong
+{
+  property: string;
+}
+
+class loooooooooooooooooooong
+  extends looooooooooooooooooong
+  implements loooooooooooooooooooong
+{
+  property: string;
+}
+
+class loooooooooooooooooooong
+  extends looooooooooooooooooong
+  implements
+    loooooooooooooooooooong,
+    loooooooooooooooooooong,
+    loooooooooooooooooooong {
```

Comment:
Why we keep `{` on the line? I think we should stick to the same style, otherwise we will have git diff when we added new `implements`